### PR TITLE
Updating to 1.8.1

### PR DESCRIPTION
**Pull request by honnibal**
First time doing this. I *think* this is all I need. Hopefully the CI helps us out :)

### CODE REVIEW

**Comment by honnibal** [comment]
Hm think I forgot to update a requirement.

**Comment by beneyal** [comment]
Yeah, `ftfy` and `regex` are missing. I was just about to try and make a PR myself, glad you beat me to it ;)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,6 +42,8 @@ requirements:
     - ujson >=1.35
     - dill >=0.2,<0.3
     - requests >=2.13.0,<3.0.0
+    - regex ==2017.4.5
+    - ftfy >=4.4.2,<5.0.0
```
Comment (rolando) [inline recipe/meta.yaml]: At this time, there is no `ftfy` package available in conda.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,6 +42,8 @@ requirements:
     - ujson >=1.35
     - dill >=0.2,<0.3
     - requests >=2.13.0,<3.0.0
+    - regex ==2017.4.5
```
Comment (rolando) [inline recipe/meta.yaml]: Even though this actually works, it may be better to match existing conda version `2017.04.05` (leading zeros).

**Comment by rolando** [comment]
`ftfy` has been merged and I've triggered a rebuild for the builds.

**Pull request merged**